avocado/utils/git.py

```diff
@@ -56,11 +57,14 @@ def __init__(
         :type base_uri: string
         :param base_uri: a closer, usually local, git repository url from where
                          to fetch content first from
+        :type submodule: string
```

Comment:
IIUIC the `submodule` should indicate if the git will use `--recursive` or not. Therefore, I would use `boolean` instead of `string`.

Reply:
@richtja Changed to boolean. Please review.
And ran the `black .` command. Thanks for the command, quick review and help.